src/main/java/gregtech/api/recipes/RecipeMap.java

```diff
@@ -236,38 +254,45 @@ protected ValidationResult<Recipe> postValidateRecipe(ValidationResult<Recipe> v
         if (recipe.isGroovyRecipe()) {
             return validationResult;
         }
-        if (!GTUtility.isBetweenInclusive(getMinInputs(), getMaxInputs(), recipe.getInputs().size())) {
-            GTLog.logger.error("Invalid amount of recipe inputs. Actual: {}. Should be between {} and {} inclusive.", recipe.getInputs().size(), getMinInputs(), getMaxInputs());
+        int amount = recipe.getInputs().size();
```

Comment:
Should there be a check here that there are inputs and that there are outputs? For example
```java
if (recipe.getInputs().size() + recipe.getFluidInputs().size() == 0 || recipe.getOutputs().size() +recipe.getChancedOutputs().size() + recipe.getFluidOutputs().size() == 0) {
        log error, no inputs or no outputs
        }
```